golang/go-cloud-run-hello-world/README.md

```diff
@@ -0,0 +1,53 @@
+# Cloud Run Hello World with Cloud Code
+
+This "Hello World" is a [Cloud Run](https://cloud.google.com/run/docs) service that renders a webpage.
+
+----
+## Table of Contents
+
+### Cloud Code for Visual Studio Code
```

Comment:
Our instructions should equally target VSCode and IntellIJ. Let's just remove this header and have "Table of contents" only.

Reply:
This is copied from other sample READMEs, I'll iterate this for Cloud Run, but maybe there are changes to make to the existing sample READMEs?

Reply:
Made some structural changes to try covering both IDEs more even-handedly. There's gaps for IntelliJ still.